Improving LLM performance for quickstarter guides. -- v1 (#149)
teamhanko/docs: quickstarts/fullstack/nuxt.mdx

## Install `@nuxtjs/hanko`

> The `@nuxtjs/hanko` module is the official Nuxt integration for Hanko authentication. It provides pre-built components (`hanko-auth`, `hanko-events`, `hanko-profile`), server middleware for route protection, and Vue composables for easy authentication management.

Reviewer: I wouldn't call it official because the module is not maintained by us.
```suggestion
Use the `@nuxtjs/hanko` module to integrate Hanko authentication. It provides pre-built components (`hanko-auth`, `hanko-events`, `hanko-profile`), server middleware for route protection, and Vue composables for easy authentication management.
```
